EIPS/eip-2771.md

```diff
@@ -1,147 +1,110 @@
 ---
 eip: 2771
 title: Secure Protocol for Native Meta Transactions
-author: Ronan Sandford (@wighawag), Liraz Siri (@lirazsiri), Dror Tirosh (@drortirosh), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Hadrien Croubois (@Amxx), Sachin Tomar (@tomarsachin2271), Patrick McCorry (@stonecoldpat), Nicolas Venturo (@nventuro), Fabian Vogelsteller (@frozeman)
+description: A contract interface for receiving meta transactions through a trusted forwarder
+author: Ronan Sandford (@wighawag), Liraz Siri (@lirazsiri), Dror Tirosh (@drortirosh), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Hadrien Croubois (@Amxx), Sachin Tomar (@tomarsachin2271), Patrick McCorry (@stonecoldpat), Nicolas Venturo (@nventuro), Fabian Vogelsteller (@frozeman), Pandapip1 (@Pandapip1)
```

Comment:
Defer to authors to decide.

Reply:
personally I am fine with this change. I am glad someone took the lead to improve the wordings with the goal to make the EIP final

Reply:
@wighawag would you mind approving this PR? https://github.com/ethereum/EIPs/pull/5607/files#submit-review